kokoro/scripts/linux/build.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# Copyright 2018 The Amber Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e  # fail on error
+set -x  # show commands
+
+BUILD_ROOT=$PWD
+SRC=$PWD/github/amber
+CONFIG=$1
+COMPILER=$2
+
+BUILD_TYPE="Debug"
+
+CMAKE_C_CXX_COMPILER=""
+if [ $COMPILER = "clang" ]
```

Comment:
Is the default one gcc?

Reply:
Seems to be the case, this is what spirv-tools does